Sources/LawParser/Data/LawSubpoint.swift

```diff
@@ -0,0 +1,8 @@
+import Vapor
+
+struct LawSubpoint: Codable {
```

Comment:
This should either conform to `Content` or remove the importing of the Vapor framework, since all the other models are Content then we should probably make this Content as well.